src/Lifecycle.ts

```diff
@@ -290,10 +285,17 @@ export interface ILocalStorageSession {
  * may not be valid, as it is not tested for consistency here.
  * @returns {Object} Information about the session - see implementation for variables.
  */
-export function getLocalStorageSessionVars(): ILocalStorageSession {
+export async function getLocalStorageSessionVars(): Promise<ILocalStorageSession> {
```

Comment:
can this method name be updated to something like `getStoredSessionVars` as its no longer all in localStorage?

Reply:
Makes sense.  Since this seems to correspond to the `persistCredentialsToLocalStorage` function, I think I'll go with something like `loadCredentials` (and also drop `ToLocalStorage` from `persistCredentialsToLocalStorage`)

Reply:
Oh, except `getLocalStorageSessionVars` and `persistCredentialsToLocalStorage` use objects that are different shapes (`homeserverUrl` vs. `hsUrl`, `guest` vs. `isGuest`, etc.).  So trying to name them similarly may cause more confusion. :-/